README.md

```diff
@@ -50,66 +50,76 @@ The project is structured around multiple components:
 
 # Getting started
 
-## Requirements
+## Platform Requirements
 
-- `python3`
-- `python3-venv`
+- The setup requires a Gen-6 or newer Intel CPU (for Intel PT) and sufficient
+  RAM to run several VMs at once.
+- A modifed Linux host kernel is required for VM-based snapshot fuzzing with
+  Intel PT coverage feedback. This setup does not run inside a VM or container!
 
-~~~
+## Local Installation
+
+The userspace installation and fuzzing workflow has been tested for recent
+Ubuntu (>=20.04) and Debian (>=bullseye). The base installation is captured
+as an Ansible workflow which you can bootstrap using Python:
```

Comment:
Does the user need to know about the underlying deployment / installation tools ?
The only requirement it to have Python3 and the capacity to create virtual environments.

Reply:
I found it misleading to say python3 is the only requirement, so I rewrote to explain we have a well-defined install flow "based on Python". Other suggestions?

We're actually pulling a lot of things as part of deployment: apt install, kernel install, grub config change, KVM group mod, kvm module options, and we don't offer uninstall. It may actually be good to document this somehow so the user has a chance to revert/debug.